test/test_integration.py

```diff
@@ -528,3 +533,22 @@ def update_graph(n_clicks):
         button.click()
         time.sleep(2)
         self.snapshot('candlestick - 2 click')
+
+    def test_graph_without_data(self):
+        app = dash.Dash(__name__)
+
+        app.layout = html.Div([
+            dcc.Graph(id='graph', figure={
+            })
+        ])
+
+        self.startServer(app=app)
+
+        graph_rendered = WebDriverWait(self.driver, 10).until(
```

Comment:
Have you tried bumping this up to like, 20?

Reply:
Yes, didn't change, should put a percy snapshot before the wait so we can see if the graph is actually there.